docs/upgrade-guide/_index.md

```diff
@@ -36,6 +37,7 @@ tail_sampling:
       dns:
         hostname: agent
         port: 4318
+  port: 4318
```

Comment:
nit: could we move the port here to be above load balancing to make it clear where it was nested? Just at a glance it's hard to tell whether it's a child of `tail_sampling` or `load_balancing` here

Reply:
Good point, moved in 2b567a273e4f62c08598a783478adb0a17dd1e40. Thanks